network/openvswitch_bridge.py

```diff
@@ -89,6 +95,19 @@
         bridge-id: "br-int"
 '''
 
+def _set_to_get(set_cmd, module):
+    """ Convert set command to get command and set value.
+    return tuple (get command, set value)
+    """
+
+    ##
+    # If set has option: then we want to truncate just before that.
+    set_cmd = truncate_before(set_cmd, " option:")
```

Comment:
`truncate_before` appears to be undefined. Here's how it was defined in `openvswitch_port` if you want to copy that over:

```python
def truncate_before(value, srch):
    """ Return content of str before the srch parameters. """

    before_index = value.find(srch)
    if (before_index >= 0):
        return value[:before_index]
    else:
        return value
```